src/melee.cpp

```diff
@@ -1369,14 +1393,6 @@ bool player::block_hit(Creature *source, body_part &bp_hit, damage_instance &dam
     // So that we don't suddenly switch that for any reason.
     const bool weapon_blocking = can_weapon_block();
     if( weapon_blocking ) {
-        int block_bonus = 2;
-        if (shield.has_technique( WBLOCK_3 )) {
-            block_bonus = 10;
-        } else if (shield.has_technique( WBLOCK_2 )) {
-            block_bonus = 6;
-        } else if (shield.has_technique( WBLOCK_1 )) {
-            block_bonus = 4;
-        }
```

Comment:
Ah. Add that in the section where that block_bonus info was removed, or where it's been moved to?